AutoRunCustomTool64/AutoRunCustomTool64.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MinimumVisualStudioVersion>17.0</MinimumVisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <ProjectGuid>{7A559F3B-752A-4FF6-A29E-E299FC9A8F59}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>AutoRunCustomTool64</RootNamespace>
+    <AssemblyName>AutoRunCustomTool64</AssemblyName>
```

Comment:
I think the name is misleading. Both projects are built for the AnyCPU platform. Why name this one "64" if it's not built for x64?
I think the names should reflect the supported VS versions. Maybe AutoRunCustomTool2015 for the old one (since it supports VS2015/2017/2019) and AutoRunCustomTool2022 for the new one?

Reply:
Yes that is much clearer. If I get time I will change the project names.

Reply:
I have renamed the projects as requested.
FYI the Target Framework of the 2015 project is .Net 4.5, which is no longer supported, and actually not easy to get from Microsoft these days.

Reply:
> FYI the Target Framework of the 2015 project is .Net 4.5, which is no longer supported, and actually not easy to get from Microsoft these days.

[Indeed](https://thomaslevesque.com/2021/11/12/building-a-project-that-target-net-45-in-visual-studio-2022/)!
Apparently VS2015 requires .NET 4.5.2, so I guess we could retarget to that. The targeting pack is still available from MS.

Reply:
ok switched to 4.5.2